flink-connector-aws/flink-connector-dynamodb/src/main/java/org/apache/flink/connector/dynamodb/source/enumerator/event/SplitsFinishedEventContext.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.flink.connector.dynamodb.source.enumerator.event;
+
+import org.apache.flink.annotation.Internal;
+
+import software.amazon.awssdk.services.dynamodb.model.Shard;
+
+import java.io.Serializable;
+import java.util.List;
+
+/** Context which contains the split id and the finished splits for a finished split event. */
+@Internal
+public class SplitsFinishedEventContext implements Serializable {
```

Comment:
This class is used as elements of a `Set`, but does not override `equals`+`hashCode`. This is an already existing bug, which I guess did not surface yet, cause we only create this set based on a `Map`, so actually there are no  real duplicates. But let's fix this.

Let's add a `serialVersionUID` constant to it too.